anndata/_core/views.py

```diff
@@ -157,6 +157,23 @@ def as_view_zappy(z, view_args):
     return z
 
 
+try:
+    from ..compat import awkward as ak
+
+    @ak.behaviors.mixins.mixin_class(ak.behavior)
+    class AwkwardArrayView(_ViewMixin, AwkArray):
+        def copy(self, order: str = "C") -> AwkArray:
+
+            return ak.copy(self)
+
+    @as_view.register(AwkArray)
+    def as_view_awkarray(array, view_args):
+        return ak.with_name(array, name="AwkwardArrayView")
+
+except ImportError:
+    pass
+
+
```

Comment:
In general, should these definitions stay where they are now or rather go into an awkward-specific file from where they are imported?